vars/CentOS-7.yml

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: MIT
```

Comment:
Make this file a symlink to RedHat-7.yml

Reply:
49792cecbec3979939bc8886b3156f71cf3b006c